src/ontology/cob-odk.yaml

```diff
@@ -10,27 +10,14 @@ workflows:
  - docs
 release_artefacts:
   - base
-  - full
-  - custom-cob-base-reasoned
-  - custom-cob-examples-reasoned
-  - custom-cob-native
+  - root
 import_group:
   module_type: filter
   annotation_properties:
     - rdfs:label
     - IAO:0000115
     - IAO:0000116
     - IAO:0000111
-  products:
-    - id: ro
-      module_type: custom
-      slme_individuals: exclude
-    - id: omo
-      module_type: mirror
-components:
-  products:
-    - filename: cob-annotations.owl
-    - filename: cob-to-external.owl
 robot_java_args: '-Xmx8G'
```

Comment:
Is the idea to hardcode whatever is needed from RO and OMO into COB templates?

Reply:
In the short-term, yes: I only wanted very specific axioms and annotations from RO and OMO.

In the medium-term we need to import terms from many ontologies into COB, very carefully. I don't know exactly what we should do.